backend/app/api/routes/auth.py

```
@@ -1,226 +1,448 @@
+            oauth_data.provider,
+            state
+        )
+
+        # Store state in session/cache for validation
+        # TODO: Store state in Redis with expiration
+
+        return {
+            "authorization_url": auth_url,
+            "state": state,
+            "provider": oauth_data.provider
+        }
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)